vkconfig/resourcefiles/Validation - Standard.json

```diff
@@ -7,15 +7,15 @@
         "layer_options": {
             "VK_LAYER_KHRONOS_validation": {
                 "debug_action": {
-                    "default": "VK_DBG_LAYER_ACTION_LOG_MSG",
+                    "default": "VK_DBG_LAYER_ACTION_DEBUG_OUTPUT",
```

Comment:
This shouldn't be changed at all, the Log Message should be the default.

Reply:
Yeah... how'd that happen... Second set of eyes is always good.